zwave_js/rootfs/etc/cont-init.d/config.sh

```diff
@@ -44,8 +45,9 @@ fi
 # Validate that no keys are using the example from the docs and generate new random
 # keys for any missing keys.
 for key in "s0_legacy_key" "s2_access_control_key" "s2_authenticated_key" "s2_unauthenticated_key"; do
-    network_key=$(bashio::string.upper "$(bashio::config \"${key}\")")
-    if [ "${network_key}" == "${DOCS_EXAMPLE_KEY_1}" ] || [ "${network_key}" == "${DOCS_EXAMPLE_KEY_2}" ] || [ "${network_key}" == "${DOCS_EXAMPLE_KEY_3}" ] || [ "${network_key}" == "${DOCS_EXAMPLE_KEY_4}" ]; then
+    network_key="$(bashio::config "${key}")"
+    network_key_upper=$(bashio::string.upper "${network_key}")
+    if [ "${network_key_upper}" == "${DOCS_EXAMPLE_KEY_1}" ] || [ "${network_key_upper}" == "${DOCS_EXAMPLE_KEY_2}" ] || [ "${network_key_upper}" == "${DOCS_EXAMPLE_KEY_3}" ] || [ "${network_key_upper}" == "${DOCS_EXAMPLE_KEY_4}" ]; then
```

Comment:
```suggestion
if [[ " ${DOCS_EXAMPLE_KEYS[*]} " =~ " ${network_key_upper} " ]]; then
```
and:
```
readonly -a DOCS_EXAMPLE_KEYS=(
        "2232666D100F795E5BB17F0A1BB7A146",
        "A97D2A51A6D4022998BEFC7B5DAE8EA1",
        "309D4AAEF63EFD85967D76ECA014D1DF",
        "CF338FE0CB99549F7C0EA96308E5A403"
)
```

For another PR